prisma/schema.prisma

```diff
@@ -99,3 +108,27 @@ model Class {
 
   @@map("classes")
 }
+
+model Information {
+  id                   String         @id @default(cuid())
+  bio                  String
+  name                 String
+  avatarUrl            String         @map("avatar_url")
```

Comment:
add `coverImageUrl `for background image in profile

Reply:
added